autoPyTorch/pipeline/base_pipeline.py

```diff
@@ -451,12 +451,14 @@ def _check_search_space_updates(self, include: Optional[Dict[str, Any]],
                             continue
                         raise ValueError("Unknown hyperparameter for component {}. "
                                          "Expected update hyperparameter "
-                                         "to be in {} got {}".format(node.__class__.__name__,
-                                                                     component.
-                                                                     get_hyperparameter_search_space(
-                                                                         dataset_properties=self.dataset_properties).
-                                                                     get_hyperparameter_names(),
-                                                                     split_hyperparameter[1]))
+                                         "to be in {} got {}."
+                                         " component is {}".format(node.__class__.__name__,
```

Comment:
What is the difference between `node.__class__.__name__` and `component.__name__`?
Both say `component`, so it would be nicer if users can know the difference from the error message.

Reply:
yeah I'll fix this comment thanks for pointing out.